manifests/kube-state-metrics-service.yaml

```diff
@@ -9,11 +9,11 @@ metadata:
 spec:
   clusterIP: None
   ports:
-  - name: http-metrics
-    port: 8080
-    targetPort: http-metrics
-  - name: telemetry
-    port: 8081
-    targetPort: telemetry
+  - name: https-main
+    port: 8443
+    targetPort: https-main
+  - name: https-self
+    port: 9443
+    targetPort: https-self
```

Comment:
do these port changes matter to us?

Reply:
Yes, we had this before merging the kube-state-metrics PR that introduced using the upstream kube-state-metrics libsonnet.

 See -> https://github.com/coreos/kube-prometheus/blob/release-0.4/manifests/kube-state-metrics-service.yaml